src/main/resources/migrations/common/V2025.05.15__public_private_comments.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE video_booking ADD COLUMN notes_for_staff VARCHAR(1000);
```

Comment:
Requests limit to 400 characters.  I have left the same length as the current comments fields.